packages/mobile/src/import/saga.ts

```diff
@@ -76,6 +77,9 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
     yield put(refreshAllBalances())
     navigate(Screens.ImportContacts)
     yield put(importBackupPhraseSuccess())
+
+    // Check if the account was verified
+    yield call(checkVerification)
```

Comment:
This must be done before the navigate call above or users could click skip before the saga is done